### PR TITLE
[Merged by Bors] - chore(Combinatorics/Additive/PluenneckeRuzsa): add @to_additive to various formulations of Ruzsa's triangle inequality

### PR DESCRIPTION
**Pull request by Command-Master**
Add `@to_additive` to `card_mul_mul_le_card_div_mul_card_div`, `card_div_mul_le_card_mul_mul_card_div` and `card_sub_mul_card_le_card_sub_mul_card_add` and correct their documentation.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean [hunk 1]
```diff
@@ -159,21 +159,27 @@ theorem card_mul_mul_card_le_card_mul_mul_card_mul (A B C : Finset α) :
 #align finset.card_mul_mul_card_le_card_mul_mul_card_mul Finset.card_mul_mul_card_le_card_mul_mul_card_mul
 #align finset.card_add_mul_card_le_card_add_mul_card_add Finset.card_add_mul_card_le_card_add_mul_card_add
 
-/-- **Ruzsa's triangle inequality**. Add-sub-sub version. -/
+/-- **Ruzsa's triangle inequality**. Mul-div-div version. -/
+@[to_additive card_add_mul_le_card_sub_mul_card_sub
+      "**Ruzsa's triangle inequality**. Add-sub-sub version."]
```
Comment (YaelDillies) [inline Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean:164]: The style is rather to not indent the additive docstring
```suggestion
"**Ruzsa's triangle inequality**. Add-sub-sub version."]
```

File: Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean [hunk 2]
```diff
@@ -159,21 +159,27 @@ theorem card_mul_mul_card_le_card_mul_mul_card_mul (A B C : Finset α) :
...
 theorem card_mul_mul_le_card_div_mul_card_div (A B C : Finset α) :
     (A * C).card * B.card ≤ (A / B).card * (B / C).card := by
   rw [div_eq_mul_inv, ← card_inv B, ← card_inv (B / C), inv_div', div_inv_eq_mul]
   exact card_mul_mul_card_le_card_mul_mul_card_mul _ _ _
 #align finset.card_mul_mul_le_card_div_mul_card_div Finset.card_mul_mul_le_card_div_mul_card_div
 
-/-- **Ruzsa's triangle inequality**. Sub-add-sub version. -/
+/-- **Ruzsa's triangle inequality**. Div-mul-div version. -/
+@[to_additive card_sub_mul_le_card_add_mul_card_sub
+      "**Ruzsa's triangle inequality**. Sub-add-sub version."]
```
Comment (YaelDillies) [inline Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean:173]:
```suggestion
"**Ruzsa's triangle inequality**. Sub-add-sub version."]
```

File: Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean [hunk 3]
```diff
@@ -159,21 +159,27 @@ theorem card_mul_mul_card_le_card_mul_mul_card_mul (A B C : Finset α) :
...
 theorem card_div_mul_le_card_mul_mul_card_div (A B C : Finset α) :
     (A / C).card * B.card ≤ (A * B).card * (B / C).card := by
   rw [div_eq_mul_inv, div_eq_mul_inv]
   exact card_mul_mul_card_le_card_mul_mul_card_mul _ _ _
 #align finset.card_div_mul_le_card_mul_mul_card_div Finset.card_div_mul_le_card_mul_mul_card_div
 
-/-- **Ruzsa's triangle inequality**. Sub-sub-add version. -/
+/-- **Ruzsa's triangle inequality**. Div-div-mul version. -/
+@[to_additive card_sub_mul_le_card_sub_mul_card_add
+      "**Ruzsa's triangle inequality**. Sub-sub-add version."]
```
Comment (YaelDillies) [inline Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean:182]:
```suggestion
"**Ruzsa's triangle inequality**. Sub-sub-add version."]
```

**Review by YaelDillies — APPROVED** [review]

File: Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean (hunk 1 above)
Comment (Command-Master) [reply Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean:164]: It's indented in the rest of the file. Should I remove that indentation or just remove it from the new `@to_additive`s?

Comment (YaelDillies) [reply Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean:164]: Uh, yes, please remove the indentation everywhere, that's a porting mistake

**Comment by YaelDillies** [comment]
Thanks!

maintainer merge

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**